core/src/com/unciv/logic/map/TileInfo.kt

```diff
@@ -205,6 +214,7 @@ open class TileInfo : IsPartOfGameInfoSerialization {
     fun isImpassible() = getLastTerrain().impassable
 
     fun getTileImprovement(): TileImprovement? = if (improvement == null) null else ruleset.tileImprovements[improvement!!]
+    fun getUnpillagedTileImprovement(): TileImprovement? = if (getUnpillagedImprovement() == null) null else ruleset.tileImprovements[improvement!!]
```

Comment:
imo if getTileImprovement is still actually in use, then *it* should be renamved getPillagedTileImprovement and getUnpillagedTileImprovement renamed  getTileImprovement. The 'default' tile improvement is an unpillaged one.

Reply:
getTileImprovement() is still used to return the tileImprovement when it doesn't matter if it's Pillaged or not. For example checking for Barbarian Camp, Ancient Ruin, and especially for Irreemovable checks or checking if there's an Improvement to Remove.

Some of that can get folded into getPillagedTileImprovement(), but I don't think getPillagedTileImprovement() would be an appropriate name for that functionality since that implies you want to get a tile improvement that's been pillaged to me